oemof/solph/blocks.py

```diff
@@ -248,16 +248,19 @@ class InvestmentFlow(SimpleBlock):
 
     **The following variables are created:**
 
-    * :math:`F_{invest}`  :attr:`om.InvestmentFlow.invest[i, o]`
+    * :attr:`invest[i, o]` - :math:`F_{invest}`
```

Comment:
I personally like this, but it is not perfectly aligned with the typical docstring, which does not name the symbol here (see e.g. https://oemof.readthedocs.io/en/stable/api/oemof.solph.html#oemof.solph.components.GenericStorage).

Reply:
That's right, it is a new suggestion for docstring style, since most of the docstrings of the basic blocks need a revision anyway to adapt to the new style (#597).

In the meantime, I thought about putting the symbols to parameters at each class as well (e.g. for investment options:  [https://oemof.readthedocs.io/en/features-add_nonconvexinvestmentflow/api/oemof.solph.html#oemof.solph.options.Investment](https://oemof.readthedocs.io/en/features-add_nonconvexinvestmentflow/api/oemof.solph.html#oemof.solph.options.Investment)), but then, I noticed that these parameter symbols might be different in each block, where they are used. E.g. `ìnvestment.maximum` can be F_{invest,max} for flows or E_{invest,max} for storages. However, I still like the idea of having the symbols there as well. If you go through the API of readthedocs, then, it would be really easy for the reader to connect the content quickly. But for sure, this makes it a lot more prone to errors and inconsistencies. What do you think about that?

To reduce dependencies of PRs and issues and some decisions, do you think it is an option to leave the docstring for now, and make it consistent before v0.0.4? I could take care of that.

Reply:
*of course I meant  v0.4.0